query_execution/QueryContext.proto

```diff
@@ -42,15 +42,20 @@ message QueryContext {
     repeated UpdateAssignment update_assignments = 2;
   }
 
+  // HashTableGroup is a vector of HashTables.
+  message HashTableGroup {
+    repeated HashTable join_hash_tables = 1;
+  }
+
   repeated AggregationOperationState aggregation_states = 1;
-  repeated BloomFilter bloom_filters = 2;
-  repeated GeneratorFunctionHandle generator_functions = 3;
-  repeated HashTable join_hash_tables = 4;
-  repeated InsertDestination insert_destinations = 5;
-  repeated Predicate predicates = 6;
-  repeated ScalarGroup scalar_groups = 7;
-  repeated SortConfiguration sort_configs = 8;
-  repeated Tuple tuples = 9;
+  repeated HashTableGroup join_hash_table_groups = 2;
+  repeated InsertDestination insert_destinations = 3;
+  repeated Predicate predicates = 4;
+  repeated ScalarGroup scalar_groups = 5;
+  repeated SortConfiguration sort_configs = 6;
+  repeated Tuple tuples = 7;
+  repeated GeneratorFunctionHandle generator_functions = 8;
+  repeated BloomFilter bloom_filters = 9;
```

Comment:
I don't understand why we move `BloomFilter` and `GeneratorFunctionHandle` to the end? We should keep all elements in the alphabet order.